python/adios2/attribute.py

```diff
@@ -58,8 +58,6 @@ def data(self):
         Returns:
             Content of the Attribute as a non string.
         """
-        if self.single_value():
```

Comment:
I intentionally added this code to return the actual value for single attributes, not an array with 1 element. So attribute reading is similar to reading variables, where single values are read as python variables or numpy 0-dim arrays